astropy/cosmology/io/tests/test_mapping.py

```diff
@@ -129,6 +129,36 @@ def test_tofrom_mapping_move_tofrom_meta(self, cosmo, to_format, from_format):
         assert got == cosmo  # (Doesn't check metadata)
         assert got.meta["mismatching"] == "will error"
 
+    def test_to_mapping_rename_conflict(self, cosmo, to_format):
+        """Test ``rename`` in ``to_mapping()``."""
+        to_rename = {"name": "name"}
+        with pytest.raises(ValueError, match="disjoint"):
```

Comment:
I would prefer if the `match` argument (here and also in other tests) contained a large enough fragment of the error message so that even someone not familiar with the code or the test suite would have a good idea what exactly is this test checking. In my experience a very detailed `match` argument helps a lot in making the test suite self-documenting.

Reply:
SGTM.

Reply:
Done=!